modules/file-integrity-node-status.adoc

```diff
@@ -3,17 +3,12 @@
 // * security/file_integrity_operator/file-integrity-operator-understanding.adoc
 
 [id="file-integrity-node-status-types_{context}"]
-= `FileIntegrityNodeStatus` status types
+= File Integrity Node Status types
```

Comment:
same comment. Should be `FileIntegrityNodeStatus` as it is a CR name.

Reply:
+1